internal/service/ipam/model_ipamsvc_host_address.go

```diff
@@ -53,38 +85,109 @@ func ExpandIpamsvcHostAddress(ctx context.Context, o types.Object, diags *diag.D
 	return m.Expand(ctx, diags)
 }
 
-func (m *IpamsvcHostAddressModel) Expand(ctx context.Context, diags *diag.Diagnostics) *ipam.IpamsvcHostAddress {
+func (m *IpamsvcHostAddressModel) Expand(_ context.Context, _ *diag.Diagnostics) *ipam.IpamsvcHostAddress {
 	if m == nil {
 		return nil
 	}
-	to := &ipam.IpamsvcHostAddress{
-		Address: flex.ExpandString(m.Address),
-		Ref:     flex.ExpandString(m.Ref),
-		Space:   flex.ExpandString(m.Space),
+	to := &ipam.IpamsvcHostAddress{}
+	
+	if !m.NextAvailableId.IsNull() && !m.NextAvailableId.IsUnknown() {
+		naip := flex.ExpandString(m.NextAvailableId) + "/nextavailableip"
+		to.Ref = &naip
```

Comment:
The Ref can also contain the UUID of an existing Address. In the model, you have made Ref as "only" computed, but it should be Optional as well.